### PR TITLE
chore: fix links to make them work well in docs.konghq.com

### PR DESCRIPTION
**Pull request by lahabana**
Also add the possibility to redact URLs from the page

### CODE REVIEW

File: app/_src/policies/introduction.md [hunk 1]
```diff
@@ -1,32 +1,30 @@
 ---
 title: Policies
 ---
-
-{% tip %}
-**Need help?** Installing and using {{site.mesh_product_name}} should be as easy as possible. [Contact and chat](/community) with the community in real-time if you get stuck or need clarifications. We are here to help.
```
Comment (slonka) [inline app/_src/policies/introduction.md:6]: why is the whole tip removed? does not seem like this is not link related

File: app/_src/policies/meshhealthcheck.md [hunk 2]
```diff
@@ -40,7 +40,7 @@ The `MeshHealthCheck` policy supports both L4/TCP and L7/HTTP/gRPC checks.
 
 ### Protocol selection
 
-The health check protocol is selected by picking the most [specific protocol](/docs/{{ page.version }}/policies/protocol-support-in-kuma/#protocol-support-in-kuma)
```
Comment (slonka) [inline app/_src/policies/meshhealthcheck.md:43]: why are we removing the fragment? maybe we could just not suffix it with "Kuma"?

File: app/_src/production/dp-config/dpp.md [hunk 3]
```diff
@@ -65,8 +65,6 @@ A `Dataplane` entity must be present for each DPP. `Dataplane` entities are mana
 - Kubernetes: The control plane {% if_version lte:2.1.x %}[**automatically generates**](/docs/{{ page.version }}/explore/dpp-on-kubernetes){% endif_version %}{% if_version gte:2.2.x %}[**automatically generates**](/docs/{{ page.version }}/production/dp-config/dpp-on-kubernetes/){% endif_version %} the `Dataplane` entity. 
 - Universal: The {% if_version lte:2.1.x %}[**user defines**](/docs/{{ page.version}}/explore/dpp-on-universal){% endif_version %}{% if_version gte:2.2.x %}[**user defines**](/docs/{{ page.version}}/production/dp-config/dpp-on-universal/){% endif_version %} the `Dataplane` entity. 
  
-For more details check the [dataplane specification](/docs/{{ page.version }}/generated/resources/proxy_dataplane):
-
```
Comment (slonka) [inline app/_src/production/dp-config/dpp.md:69]: again whole section removed

File: app/_src/production/cp-deployment/multi-zone.md [hunk 4]
```diff
@@ -419,7 +419,7 @@ curl http://echo-server:1010
 Requests are distributed round robin between zones.
 You can use [locality-aware load balancing](/docs/{{ page.version }}/policies/locality-aware) to keep requests in the same zone.
 
-To send a request to any zone, you can {% if_version lte:2.1.x %}[use the generated `kuma.io/service`](/docs/{{ page.version }}/explore/dpp-on-kubernetes#tag-generation){% endif_version %}{% if_version gte:2.2.x %}[use the generated `kuma.io/service`](/docs/{{ page.version }}/production/dp-config/dpp-on-kubernetes/#tag-generation){% endif_version %} and [{{site.mesh_product_name}} DNS](/docs/{{ page.version }}/networking/dns#dns):
```
Comment (slonka) [inline app/_src/production/cp-deployment/multi-zone.md:422]: url fragment removed here and probably in other places

File: app/_src/policies/introduction.md (hunk 1 above)
Comment (lahabana) [reply app/_src/policies/introduction.md:6]: It's useless and hard to maintain

File: app/_src/policies/meshhealthcheck.md (hunk 2 above)
Comment (lahabana) [reply app/_src/policies/meshhealthcheck.md:43]: This fragment doesn't exist (it's just the page)

File: app/_src/production/dp-config/dpp.md (hunk 3 above)
Comment (lahabana) [reply app/_src/production/dp-config/dpp.md:69]: reference is inline now no need for the link

**Pull request merged**